bigframes/display/__init__.py

```diff
@@ -16,11 +16,24 @@
 
 from __future__ import annotations
 
-try:
-    import anywidget  # noqa
+from typing import Any
 
-    from bigframes.display.anywidget import TableWidget
 
-    __all__ = ["TableWidget"]
-except Exception:
-    pass
+def __getattr__(name: str) -> Any:
```

Comment:
This isn't typical to do, so could you please add some comments explaining why we defer in this way in the docstring and/or comments as well?